frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -49,11 +102,23 @@ class AccountDetails extends React.Component<Props> {
                   text={
                     <>
                       <span>
-                        &uarr;{account.outTransactionsCount.toLocaleString()}
+                        &uarr;
+                        {outTransactionsCount &&
+                        typeof outTransactionsCount !== undefined ? (
```

Comment:
1. `(0 && ...)` is `false`, so the check is incorrect
2. using `typeof` you will get a string `"undefined"`

Thus, here is the proper check:

```suggestion
                        {outTransactionsCount !== undefined ? (
```